core/field_colour.js

```diff
@@ -611,32 +611,32 @@ FieldColour.prototype.dropdownDispose_ = function() {
 Css.register([
```

Comment:
Do we want to revert this, or find some way to not have to fight clang on it? Applies throughout as there are a few places where CSS is registered and it wants to dedent it.